packages/components/src/passengerCards/AddPassengerButton.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Icon, Button, StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +onPress: () => void,
+|};
+
+export default function AddPassengerButton({ onPress }: Props) {
+  return (
+    <View style={styles.addPassengerAndroid}>
```

Comment:
I think this wrapper and its style is unnecessary.
It seems that it is required only to solve problem with oval button on Android. But we will probably need to fix this problem directly inside `Button` component (it seems that width prop currently only affects `Touchable` inside button which is probably not enough for Android version).

I suggest to remove this wrapper and create separate issue for oval circled button on Android.

Reply:
Done (issue here: https://github.com/kiwicom/margarita/issues/528)